cypress/integration/smoke/RESTInteractionTest.ts

```diff
@@ -0,0 +1,117 @@
+import { onlyOn } from "@cypress/skip-test";
+import {
+  EnvType,
+  isEnvironmentType,
+  safeLogin,
+  uniqueName,
+} from "../../utils/Util";
+
+onlyOn(isEnvironmentType(EnvType.Dev), () => {
+  describe("UI react on the REST interactions", () => {
+    const bridgeName = uniqueName("myBridge");
+    const restBaseUrl = Cypress.env("SANDBOX_DEV_REST_URL");
+    const restPath = Cypress.env("SANDBOX_DEV_REST_PATH");
+    const restUrl = `${restBaseUrl}${restPath}/bridges`;
+    const token = Cypress.env("OB_TOKEN");
+    const user: string = Cypress.env("USER");
+    const psw: string = Cypress.env("PASSWORD");
+    const authorization = `Bearer ${token}`;
+    let bridgeId: String;
+
+    before(() => {
+      if (!token) {
+        throw new Error("Missing token value, set using CYPRESS_OB_TOKEN");
+      } else {
+        cy.log("Token is set correctly");
+      }
+
+      expect(restBaseUrl, "REST base url was set").to.be.a("string").and.not.be
+        .empty;
+      expect(restPath, "REST path was set").to.be.a("string").and.not.be.empty;
+
+      if (typeof user !== "string" || !user) {
+        throw new Error("Missing user value, set using CYPRESS_USER");
+      } else {
+        cy.log("User name is set correctly");
+      }
+
+      if (typeof psw !== "string" || !psw) {
+        throw new Error("Missing password value, set using CYPRESS_PASSWORD");
+      } else {
+        cy.log("Password is set correctly");
+      }
+
+      const options = {
+        method: "POST",
+        url: restUrl,
+        headers: {
+          "Content-Type": "application/json",
+          Accept: "application/json",
+          Authorization: authorization,
+        },
+        body: {
+          name: bridgeName,
+          cloud_provider: "aws",
+          region: "us-east-1",
+        },
+      };
+      cy.request(options).then((response) => {
+        // response.body is automatically serialized into JSON
+        expect(response.body).to.not.be.null;
+        expect(response.body.id).to.not.be.null;
+        bridgeId = response.body.id;
+      });
+    });
```

Comment:
@jomarko I hope that I undrestand the question correctly.
Current state:
- no pre-condition -> run on both environment (Mocked and DEV)
- pre-condition -> define which environment is a target (Mocked xor DEV)

Reply:
@sjamboro yes, I understand it as you describe.

However I think `BasicTest` is running on both mocked and DEV, so I am not sure why we do not need there this "TOKEN and variables initialization part" that we have in `RESTInteractionTest`

Reply:
OK ... I will check it. `token` is only for REST. I should check `user` and `password`.

Reply:
I do not think that we need `if (typeof user !== "string" || !user) {..}` check for all test but once it is helpfull. Because it read ENV and I suppose that it is stable for all executions. Maybe move this checks under a new test in Basic test running only with DEV. The name should be `Check environment`.

Reply:
I think what you mention is OK for CI where we run all tests, however for local development and running just single tets usually, I am not sure if user will be informed what variables were not set

Reply:
@jomarko I have posponed it to https://issues.redhat.com/browse/MGDOBR-1041

Reply:
thank you @sjamboro